src/tests/test_CA/SSSD_test_cert_0005.config

```diff
@@ -0,0 +1,21 @@
+# This certificate is used in
+# - src/tests/cmocka/test_pam_srv.c
+# as an example with a OCSP URI
+[ req ]
+distinguished_name = req_distinguished_name
+prompt = no
+
+[ req_distinguished_name ]
+O = SSSD
+OU = SSSD test
+CN = SSSD test cert 0005
+
+[ req_exts ]
```

Comment:
Have we missed crlDistributionPoints?

Reply:
No, since SSSD does not try to load CRLs from external resources but relies on other processes to update the CRLs.